pylint_django/checkers/migrations.py

```diff
@@ -0,0 +1,41 @@
+import astroid
+from pylint import interfaces
+from pylint.checkers import utils, BaseChecker
+from pylint_django.__pkginfo__ import BASE_ID
+
+def _is_migrations_module(node):
+    if not isinstance(node, astroid.Module):
+        return False
+
+    return 'migrations' in node.path[0] and not node.path[0].endswith('__init__.py')
+
+
+class MissingBackwardsMigrationChecker(BaseChecker):
+    __implements__ = (interfaces.IAstroidChecker,)
+
+    name = 'missing-backwards-migration-checker'
+
+    msgs = {'W%s05' % BASE_ID: ('Always include backwards migration callable',
+                                'missing-backwards-migration-callable',
+                                'Always include a backwards/reverse callable counterpart'
+                                ' so that the migration is not irreversable.')}
+
+    def visit_call(self, node):
+        try:
+            module = node.frame().parent
+        except:  # noqa: E722, pylint: disable=bare-except
+            return
+
+        if not _is_migrations_module(module):
+            return
+
+        if node.func.as_string().endswith('RunPython') and len(node.args) < 2:
```

Comment:
Don't use `as_string()`, function calls have attribute which has a name which matches `RunPython`.

Reply:
I had tried `node.func.attrname` but I was getting AttributeError's. I think we might run into the same issue as [here](https://github.com/kiwitcms/Kiwi/pull/1742#discussion_r441485689) if we rely on the attribute of the node to give us the name. We would have to assume that `RunPython` isn't being imported directly as opposed to using the whole migrations import.

Reply:
This check needs to be more strict. Use isinstance() to make sure that node.func is an Attribute and then you can safely access node.func.attrname

Reply:
But what if 
`from django.db.migrations import RunPython` is used?
wouldn't `RunPython(forwards, backwards)` be a `Name` node type? and be missed by the winter?

Reply:
One slight problem we've run into before is that any imports from `django.db` make django try to configure the DB connection and this can fail, especially when Django isn't installed for CI jobs (which happens annoyingly more than you'd expect).

Reply:
> One slight problem we've run into before is that any imports from django.db make django try to configure the DB connection and this can fail, especially when Django isn't installed for CI jobs (which happens annoyingly more than you'd expect).

@carlio you have misread the comment from @brymut. He meant that when the migration imports RunPython as `from django.db.migrations import RunPython` it changes the astroid node type to Name (because it is a method/class name) and when it's used as `migrations.RunPython` it is an attribute b/c we're accessing an attribute of the migrations module.

Reply:
@brymut agreed, let's leave the as_string() comparison although I don't like it. 

FTR in the reference that you posted the string comparison has an unwanted side effect because forms, models and admin classes can have fields whose name ends with `Field`. And their types are different hence the broader match. 

For RunPython this is less so.